packages/material-ui/src/Select/Select.test.js

```diff
@@ -64,7 +63,149 @@ describe('<Select />', () => {
     expect(container.querySelector('input')).to.have.property('value', '10');
   });
 
+  specify('the trigger is in tab order', () => {
```

Comment:
> Similarly, specify() is an alias for it().

https://mochajs.org/#bdd

```suggestion
  it('the trigger is in tab order', () => {
```

Reply:
I get that the name is bad (jest uses `test`) but `it` is worse.

Reply:
Could we prioritize the API surface rather than the readability? For instance, Jest only allows one method name: `test`. Should it be enough?

Reply:
Would it make sense to us our own alias? test and to deploy it everywhere?

Reply:
> Jest only allows one method name: test.

Jest has `it` and `test` as well.

> Which one should I use?

The one that is a valid sentence in english language. 

> When, at the end of the day, it won't make a difference.

Then why start a discussion about it if one has less readability and one more?

Reply:
I can't find the `it` in the Jest documentation. Did they burry it? https://jestjs.io/docs/en/api#methods